src/app/components/configuration/configuration.component.ts

```diff
@@ -175,7 +176,9 @@ export class ConfigurationComponent implements CourseConfigurable {
                 )}\n***ACTION NEEDED***: \nYou can use Canvas to delete the Token ATM content by manually deleting: \n1) the two Canvas pages prefixed with 'Token ATM', \n2) the one Canvas assignment group prefixed with 'Token ATM', and \n3) the one Canvas module prefixed with 'Token ATM'. \n***Sorry for the inconvenience!`,
```

Comment:
The location of `***ACTION NEEDED***` information is not consistent with `onResetContent()` and `onMigrate()`: it should be above the serialized error.

Reply:
It seems that this change is included in #60, so please dismiss this.